src/quacc/recipes/newtonnet/ts.py

```diff
@@ -42,6 +45,19 @@ class QuasiIRCSchema(OptSchema):
         irc_job: IRCSchema
         freq_job: FreqSchema | None
 
+    class NebSchema(TypedDict):
+        relax_reactant: OptSchema
+        relax_product: OptSchema
+        geodesic_results: list[Atoms]
+        neb_results: dict
+
+    class NebTsSchema(TypedDict):
+        relax_reactant: OptSchema
+        relax_product: OptSchema
+        geodesic_results: list[Atoms]
+        neb_results: dict
+        ts_results: TSSchema
```

Comment:
The `neb_results: dict` should use the type hint for the return from `summarize_neb_run` (if we keep that function) rather than `dict`.

Reply:
Not sure how to do this one. I see that quacc.types has the base classes to define the schemas but I don't have those for summarize_neb_run.